demos/python_demos/human_pose_estimation_demo/README.md

```diff
@@ -0,0 +1,45 @@
+# Human Pose Estimation Demo (top-down pipeline)
+
+This demo showcases top-down pipeline for human pose estimation on video or image. The task is to predict bboxes for everyone person on frame and then to predict a pose for everyone detected person. The pose may contain up to 17 keypoints: ears, eyes, nose, shoulders, elbows, wrists, hips, knees, and ankles.
+
+# How It Works
+
+On the start-up, the application reads command line parameters and loads  detection person model and single human pose estimation model. Upon getting a frame from the OpenCV VideoCapture, then  demo executes top/down pipeline for this frame and displays the results.
+
+# Running
+
+Running the application with the `-h` option yields the following usage message:
+```
+usage: demo.py [-h] --model-od-xml MODEL_OD_XML --model-od-bin MODEL_OD_BIN
+               --model-hpe-xml MODEL_HPE_XML --model-hpe-bin MODEL_HPE_BIN
+               [--video VIDEO] [--image IMAGE [IMAGE ...]] [--device DEVICE]
+               [--cpu-extension CPU_EXTENSION] [--label-person LABEL_PERSON]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --model-od-xml MODEL_OD_XML
+                        path to model of object detector in xml format
+  --model-od-bin MODEL_OD_BIN
+                        path to model of object detector in bin format
+  --model-hpe-xml MODEL_HPE_XML
+                        path to model of human pose estimator in xml format
+  --model-hpe-bin MODEL_HPE_BIN
+                        path to model of human pose estimator in bin format
+  --video VIDEO         path to video
+  --image IMAGE [IMAGE ...]
+                        path to image or images
+  --device DEVICE       Specify the target to infer on CPU or GPU
+  --cpu-extension CPU_EXTENSION
+                        path to cpu extension
+  --label-person LABEL_PERSON
+                        Label of class person for detector
```

Comment:
Please make all keys according to [contribution guide](https://github.com/opencv/open_model_zoo/blob/develop/CONTRIBUTING.md#demo) (e.g. `-m_*` for model, add `--no-show` key, etc)

Reply:
@eizamaliev done